diff --git a/bin/account.php b/bin/account.php
index 579070bd..a6d44657 100644
--- a/bin/account.php
+++ b/bin/account.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use Vpn\Portal\Cfg\Config;
diff --git a/bin/status.php b/bin/status.php
index 4f4685f9..759efb40 100644
--- a/bin/status.php
+++ b/bin/status.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Cfg\Config;
 use Vpn\Portal\Cfg\ProfileConfig;
diff --git a/libexec/daemon-sync.php b/libexec/daemon-sync.php
index 7c9d2857..afd92cab 100644
--- a/libexec/daemon-sync.php
+++ b/libexec/daemon-sync.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Cfg\Config;
 use Vpn\Portal\HttpClient\CurlHttpClient;
diff --git a/libexec/db.php b/libexec/db.php
index 92a60f4d..811512cd 100644
--- a/libexec/db.php
+++ b/libexec/db.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Cfg\Config;
 use Vpn\Portal\Migration;
diff --git a/libexec/generate-secrets.php b/libexec/generate-secrets.php
index d67a921d..5a3c4027 100644
--- a/libexec/generate-secrets.php
+++ b/libexec/generate-secrets.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\Signer;
 use Vpn\Portal\Cfg\Config;
diff --git a/libexec/housekeeping.php b/libexec/housekeeping.php
index 6f6be26f..c5643df8 100644
--- a/libexec/housekeeping.php
+++ b/libexec/housekeeping.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Cfg\Config;
 use Vpn\Portal\Dt;
diff --git a/libexec/stats.php b/libexec/stats.php
index d115580c..c47c3a63 100644
--- a/libexec/stats.php
+++ b/libexec/stats.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Cfg\Config;
 use Vpn\Portal\ConnectionManager;
diff --git a/web/api.php b/web/api.php
index 5fee3626..10df4652 100644
--- a/web/api.php
+++ b/web/api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/index.php b/web/index.php
index d8e73923..e06222f0 100644
--- a/web/index.php
+++ b/web/index.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/node-api.php b/web/node-api.php
index 7b650751..225453bd 100644
--- a/web/node-api.php
+++ b/web/node-api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Cfg\Config;
 use Vpn\Portal\Http\Auth\NodeAuthModule;
diff --git a/web/oauth.php b/web/oauth.php
index 2a6d00ee..b0f1ebcc 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/well-known.php b/web/well-known.php
index f2b9913f..ccd04872 100644
--- a/web/well-known.php
+++ b/web/well-known.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\FileIO;
 use Vpn\Portal\Http\JsonResponse;
