diff --git a/bin/add-user.php b/bin/add-user.php
index 6b5dc84..94e69b1 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\Config;
 use LC\Portal\Storage;
diff --git a/bin/generate-dns-zones.php b/bin/generate-dns-zones.php
index aa848fd..74b44c1 100644
--- a/bin/generate-dns-zones.php
+++ b/bin/generate-dns-zones.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\Config;
 use LC\Portal\IP;
diff --git a/bin/show-oauth-key.php b/bin/show-oauth-key.php
index 4bdfcbc..dc22213 100644
--- a/bin/show-oauth-key.php
+++ b/bin/show-oauth-key.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Portal\FileIO;
diff --git a/libexec/foreign-key-list-fetcher.php b/libexec/foreign-key-list-fetcher.php
index 19c4b6e..ceface8 100644
--- a/libexec/foreign-key-list-fetcher.php
+++ b/libexec/foreign-key-list-fetcher.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\Config;
 use LC\Portal\Federation\ForeignKeyListFetcher;
diff --git a/libexec/generate-secrets.php b/libexec/generate-secrets.php
index 68a7bb4..3401e24 100644
--- a/libexec/generate-secrets.php
+++ b/libexec/generate-secrets.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Portal\FileIO;
diff --git a/libexec/housekeeping.php b/libexec/housekeeping.php
index 57512fe..15a93f2 100644
--- a/libexec/housekeeping.php
+++ b/libexec/housekeeping.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\Config;
 use LC\Portal\Storage;
diff --git a/libexec/init.php b/libexec/init.php
index 9667645..b34dd2b 100644
--- a/libexec/init.php
+++ b/libexec/init.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\Config;
 use LC\Portal\FileIO;
diff --git a/libexec/stats.php b/libexec/stats.php
index 1a15d4a..ce890e3 100644
--- a/libexec/stats.php
+++ b/libexec/stats.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\Config;
 use LC\Portal\FileIO;
diff --git a/libexec/wg-sync-peers.php b/libexec/wg-sync-peers.php
index 7582d40..7ff695c 100644
--- a/libexec/wg-sync-peers.php
+++ b/libexec/wg-sync-peers.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\Config;
 use LC\Portal\HttpClient\CurlHttpClient;
diff --git a/web/api.php b/web/api.php
index 8c1ee13..f9ff2ce 100644
--- a/web/api.php
+++ b/web/api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Portal\CA\VpnCa;
diff --git a/web/index.php b/web/index.php
index 8bd16f9..30db5ee 100644
--- a/web/index.php
+++ b/web/index.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use fkooman\SeCookie\Cookie;
diff --git a/web/node-api.php b/web/node-api.php
index d525c6a..167e6b6 100644
--- a/web/node-api.php
+++ b/web/node-api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\CA\VpnCa;
 use LC\Portal\Config;
diff --git a/web/oauth.php b/web/oauth.php
index 5fb35dc..db906f0 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Portal\Config;
diff --git a/web/well-known.php b/web/well-known.php
index 7ee1059..88bc8c4 100644
--- a/web/well-known.php
+++ b/web/well-known.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Portal\FileIO;
 use LC\Portal\Http\JsonResponse;
