diff --git a/bin/add-user.php b/bin/add-user.php
index 47a77a2b..93d757c2 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Dt;
diff --git a/bin/generate-dns-zones.php b/bin/generate-dns-zones.php
index 909bc328..803c70ab 100644
--- a/bin/generate-dns-zones.php
+++ b/bin/generate-dns-zones.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Binary;
 use Vpn\Portal\Config;
diff --git a/bin/status.php b/bin/status.php
index 067e94fa..a01dc93d 100644
--- a/bin/status.php
+++ b/bin/status.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\ConnectionManager;
diff --git a/libexec/daemon-sync.php b/libexec/daemon-sync.php
index d0467612..67041caa 100644
--- a/libexec/daemon-sync.php
+++ b/libexec/daemon-sync.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\HttpClient\CurlHttpClient;
diff --git a/libexec/generate-secrets.php b/libexec/generate-secrets.php
index 3e1b1da4..0daa4325 100644
--- a/libexec/generate-secrets.php
+++ b/libexec/generate-secrets.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\Signer;
 use Vpn\Portal\FileIO;
diff --git a/libexec/housekeeping.php b/libexec/housekeeping.php
index c9a0152e..3d2682fd 100644
--- a/libexec/housekeeping.php
+++ b/libexec/housekeeping.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Dt;
diff --git a/libexec/init.php b/libexec/init.php
index cac1d894..faa2e542 100644
--- a/libexec/init.php
+++ b/libexec/init.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\FileIO;
diff --git a/web/api.php b/web/api.php
index 2145bb29..4d634423 100644
--- a/web/api.php
+++ b/web/api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\BearerValidator;
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
diff --git a/web/index.php b/web/index.php
index 1c5d2403..94e5bb85 100644
--- a/web/index.php
+++ b/web/index.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/node-api.php b/web/node-api.php
index 0a550670..75b04ad8 100644
--- a/web/node-api.php
+++ b/web/node-api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\FileIO;
diff --git a/web/oauth.php b/web/oauth.php
index 69ef5e5d..ace4a1ea 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/well-known.php b/web/well-known.php
index c326341b..c1eadd09 100644
--- a/web/well-known.php
+++ b/web/well-known.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\FileIO;
 use Vpn\Portal\Http\JsonResponse;
