diff --git a/bin/add-user.php b/bin/add-user.php
index 63c5177..d7a5b72 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Common\Config;
 use LC\Portal\Storage;
diff --git a/bin/show-oauth-key.php b/bin/show-oauth-key.php
index 9e92551..0d7d8c3 100644
--- a/bin/show-oauth-key.php
+++ b/bin/show-oauth-key.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Common\FileIO;
diff --git a/libexec/foreign-key-list-fetcher.php b/libexec/foreign-key-list-fetcher.php
index 9ee216e..89bf665 100644
--- a/libexec/foreign-key-list-fetcher.php
+++ b/libexec/foreign-key-list-fetcher.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Common\Config;
 use LC\Common\HttpClient\CurlHttpClient;
diff --git a/libexec/generate-secrets.php b/libexec/generate-secrets.php
index 1036547..4ec9c3b 100644
--- a/libexec/generate-secrets.php
+++ b/libexec/generate-secrets.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\Jwt\Keys\EdDSA\SecretKey;
 use LC\Common\FileIO;
diff --git a/libexec/init.php b/libexec/init.php
index 3e6d6d9..3463c19 100644
--- a/libexec/init.php
+++ b/libexec/init.php
@@ -7,8 +7,8 @@
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/LC/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use LC\Common\FileIO;
 use LC\Portal\Storage;
