diff --git a/bin/add-user.php b/bin/add-user.php
index e879b6b9..15e2c30b 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Dt;
diff --git a/bin/generate-dns-zones.php b/bin/generate-dns-zones.php
index 909bc328..803c70ab 100644
--- a/bin/generate-dns-zones.php
+++ b/bin/generate-dns-zones.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Binary;
 use Vpn\Portal\Config;
diff --git a/bin/status.php b/bin/status.php
index 044b3ae3..be7c9341 100644
--- a/bin/status.php
+++ b/bin/status.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\ConnectionManager;
diff --git a/libexec/daemon-sync.php b/libexec/daemon-sync.php
index 13c5ea3d..6047f820 100644
--- a/libexec/daemon-sync.php
+++ b/libexec/daemon-sync.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\HttpClient\CurlHttpClient;
diff --git a/libexec/generate-secrets.php b/libexec/generate-secrets.php
index 7b177359..9dbfe95f 100644
--- a/libexec/generate-secrets.php
+++ b/libexec/generate-secrets.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\Signer;
 use Vpn\Portal\Config;
diff --git a/libexec/housekeeping.php b/libexec/housekeeping.php
index d535753c..4dc132d0 100644
--- a/libexec/housekeeping.php
+++ b/libexec/housekeeping.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Dt;
diff --git a/libexec/init-db.php b/libexec/init-db.php
index 7f95a2bf..918d75a1 100644
--- a/libexec/init-db.php
+++ b/libexec/init-db.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Storage;
diff --git a/web/api.php b/web/api.php
index 8fd46e32..09735939 100644
--- a/web/api.php
+++ b/web/api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\BearerValidator;
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
diff --git a/web/index.php b/web/index.php
index 7aa6cde6..4e4db803 100644
--- a/web/index.php
+++ b/web/index.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/node-api.php b/web/node-api.php
index f9d201f5..aba7fb64 100644
--- a/web/node-api.php
+++ b/web/node-api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\FileIO;
diff --git a/web/oauth.php b/web/oauth.php
index 6ffedbd3..6784daf6 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/well-known.php b/web/well-known.php
index c326341b..c1eadd09 100644
--- a/web/well-known.php
+++ b/web/well-known.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\FileIO;
 use Vpn\Portal\Http\JsonResponse;
