diff --git a/bin/add-user.php b/bin/add-user.php
index 36da84f6..0f770f56 100644
--- a/bin/add-user.php
+++ b/bin/add-user.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Dt;
diff --git a/bin/status.php b/bin/status.php
index b0d92ffe..2ed76ae3 100644
--- a/bin/status.php
+++ b/bin/status.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\ConnectionManager;
diff --git a/libexec/daemon-sync.php b/libexec/daemon-sync.php
index e2fe9915..3eb820f2 100644
--- a/libexec/daemon-sync.php
+++ b/libexec/daemon-sync.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\HttpClient\CurlHttpClient;
diff --git a/libexec/db.php b/libexec/db.php
index a2d23691..b1fb8dbf 100644
--- a/libexec/db.php
+++ b/libexec/db.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Migration;
diff --git a/libexec/generate-secrets.php b/libexec/generate-secrets.php
index e9f69440..b2a4cee9 100644
--- a/libexec/generate-secrets.php
+++ b/libexec/generate-secrets.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\Signer;
 use Vpn\Portal\Config;
diff --git a/libexec/housekeeping.php b/libexec/housekeeping.php
index b692a120..005394a3 100644
--- a/libexec/housekeeping.php
+++ b/libexec/housekeeping.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Dt;
diff --git a/libexec/stats.php b/libexec/stats.php
index 06dd5f1f..ab35d28d 100644
--- a/libexec/stats.php
+++ b/libexec/stats.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\ConnectionManager;
diff --git a/web/api.php b/web/api.php
index c4ea9fad..f3ec5a0f 100644
--- a/web/api.php
+++ b/web/api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/index.php b/web/index.php
index 44e8d550..f48d05ec 100644
--- a/web/index.php
+++ b/web/index.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/node-api.php b/web/node-api.php
index 674f68fb..848e5d0e 100644
--- a/web/node-api.php
+++ b/web/node-api.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\Config;
 use Vpn\Portal\Http\Auth\NodeAuthModule;
diff --git a/web/oauth.php b/web/oauth.php
index b8290182..3c006195 100644
--- a/web/oauth.php
+++ b/web/oauth.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use fkooman\OAuth\Server\PdoStorage as OAuthStorage;
 use fkooman\OAuth\Server\Signer;
diff --git a/web/well-known.php b/web/well-known.php
index f2b9913f..ccd04872 100644
--- a/web/well-known.php
+++ b/web/well-known.php
@@ -9,8 +9,8 @@ declare(strict_types=1);
  * SPDX-License-Identifier: AGPL-3.0+
  */
 
-require_once dirname(__DIR__).'/vendor/autoload.php';
-$baseDir = dirname(__DIR__);
+require_once '/usr/share/php/Vpn/Portal/autoload.php';
+$baseDir = '/usr/share/vpn-user-portal';
 
 use Vpn\Portal\FileIO;
 use Vpn\Portal\Http\JsonResponse;
